src/components/book.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable react/destructuring-assignment */
```

Comment:
- [ ] Please, consider removing the linter disables, let the linter does it`s direct job - find the bugs on the early steps of you project